styles/facts.css

```diff
@@ -0,0 +1,40 @@
+body {
+    /* font-family: Arial, sans-serif; */
```

Comment:
Don't forget to make sure that you delete the commented out code to help keep your code base clutter free.